diff -Naur syslinux-1.71/syslinux.c syslinux-1.71-p/syslinux.c
--- syslinux-1.71/syslinux.c	Thu Apr 18 07:05:12 2002
+++ syslinux-1.71-p/syslinux.c	Thu Apr 18 12:50:48 2002
@@ -102,7 +102,7 @@
 
 void usage(void)
 {
-  fprintf(stderr, "Usage: %s [-sf] [-o offset] device\n", program);
+  fprintf(stderr, "Usage: %s [-sfn] [-o offset] device\n", program);
   exit(1);
 }
 
@@ -180,7 +180,8 @@
   int my_umask;
   int force = 0;		/* -f (force) option */
   off_t offset = 0;		/* -o (offset) option */
-
+  int no_mount = 0;		/* -n (no_mount) option */
+  
   program = argv[0];
   
   device = NULL;
@@ -196,6 +197,8 @@
 	  make_stupid();	/* Use "safe, slow and stupid" code */
 	} else if ( *opt == 'f' ) {
 	  force = 1;		/* Force install */
+	} else if ( *opt == 'n' ) {
+	  no_mount = 1;		/* Do not mount device/image to put ldlinux.sys */
 	} else if ( *opt == 'o' && argp[1] ) {
 	  offset = strtoul(*++argp, NULL, 0); /* Byte offset */
 	} else {
@@ -301,6 +304,9 @@
 	    device);
   }
 
+
+  if (!no_mount) {
+  
   /*
    * Now mount the device.  If we are non-root we need to find an fstab
    * entry for this device which has the user flag and the appropriate
@@ -455,6 +461,10 @@
   if ( err )
     exit(err);
 
+
+  } /* end of if (!no_mount) */
+
+  
   /*
    * To finish up, write the boot sector
    */
diff -Naur syslinux-1.71/syslinux.doc syslinux-1.71-p/syslinux.doc
--- syslinux-1.71/syslinux.doc	Thu Apr 18 07:05:12 2002
+++ syslinux-1.71-p/syslinux.doc	Thu Apr 18 12:50:24 2002
@@ -75,6 +75,13 @@
 The -o option is used with a disk image file and specifies the byte
 offset of the filesystem image in the file.
 
+The -n option can be used to disable mounting floppy device/image.
+In normal mode syslinux tries to mount filesystem for a moment
+to copy ldlinux.sys onto it. Alternatively, user can put ldlinux.sys 
+(e.g with mtools) and then run syslinux with -n option.
+This enables creation of bootdisks by ordinary users in case they are not 
+allowed to mount anything.
+
 On boot time, by default, the kernel will be loaded from the image named
 LINUX on the boot floppy.  This default can be changed, see the section
 on the SYSLINUX config file.
