--- syslinux-2.04/memdisk/init.S16	Thu Apr 10 06:28:08 2003
+++ syslinux-2.04/memdisk/init.S16-	Thu May 22 23:56:54 2003
@@ -42,7 +42,7 @@
 _start:
 		jmp	start
 
-# This is the setup header, and it must start at %cs:2 (old 0x9020:2)
+/* This is the setup header, and it must start at %cs:2 (old 0x9020:2) */
 	
 		.ascii	"HdrS"		# header signature
 		.word	0x0203		# header version number (>= 0x0105)
@@ -57,7 +57,7 @@
 					# See Documentation/i386/boot.txt for
 					# assigned ids
 	
-# flags, unused bits must be zero (RFU) bit within loadflags
+/* flags, unused bits must be zero (RFU) bit within loadflags */
 loadflags:
 LOADED_HIGH	= 1			# If set, the kernel is loaded high
 CAN_USE_HEAP	= 0x80			# If set, the loader also has set
